sig-ops/README.md

```diff
@@ -0,0 +1,27 @@
+# SIG: Ops
+
+This Special Interest Group (SIG) main goal is to coordinate Operations (Ops) activities and best practices across Spinnaker Operators.
```

Comment:
How about this instead:

_The Operations (Ops) Special Interest Group (SIG) provides a forum for groups that are operating Spinnaker deployments to coordinate their activities and identify and share best practices._

Reply:
sounds good! I will update it.